spec/checkPullRequestReviewSpec.js

```diff
@@ -1522,5 +1522,26 @@ describe('Pull Request Review Module', () => {
         });
       }
     );
+
+    describe(
+      'when PTAL template found in the comment',
+      () => {
+        const initialCommentBody = commentPayloadData.payload.comment.body;
+        beforeAll(() => {
+          commentPayloadData.payload.comment.body =
+          '{{Question/comment}} @{{reviewer_username}} PTAL';
```

Comment:
```suggestion
          commentPayloadData.payload.comment.body = (
          '{{Question/comment}} @{{reviewer_username}} PTAL');
```
Always break after parenthesis.

Reply:
done